### PR TITLE
LPS-114982

### PR DESCRIPTION
**Pull request by RitaSilveira**

### CODE REVIEW

**Comment by RitaSilveira** [comment]
@CarlosBrichete

**Comment by RitaSilveira** [comment]
https://issues.liferay.com/browse/LPS-133339 , cc: @CarlosBrichete

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 1]
```diff
@@ -1564,4 +1564,80 @@ definition {
...
+
+        ApplicationsMenu.gotoPortlet(
+            category = "Sites",
+            panel = "Control Panel",
+            portlet = "Sites");
+
+        Site.addBlankCP(
+            membershipType = "Private",
+            siteName = "Blank Site Name");
+
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase]: Use the JSON macro to add the new site. See: https://issues.liferay.com/browse/LPS-130691

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 2]
```diff
@@ -1564,4 +1564,80 @@ definition {
...
+        ProductMenu.gotoPortlet(
+            category = "Site Builder",
+            portlet = "Pages");
+
+        SitePages.addPublicPage(
+            pageName = "Site Page Name",
+            sitePageType = "Widget Page");
+
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase]: Use the JSON macro to add the new paga in the Site. See: https://issues.liferay.com/browse/LPS-130691

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 3]
```diff
@@ -1564,4 +1564,80 @@ definition {
...
+        Navigator.gotoSitePage(
+            pageName = "Site Page Name",
+            siteName = "Blank Site Name");
+
+        MessageboardsThread.addPGWithPermissions(
+            threadBody = "MB Thread Message Body",
+            threadSubject = "MB Thread Message Subject",
+            viewableBySelection = "Owner");
+
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase]: Why is this MessageboardsThread needed?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 4]
```diff
@@ -1564,4 +1564,80 @@ definition {
...
+            category = "Users",
+            panel = "Control Panel",
+            portlet = "Users and Organizations");
+			
+        User.addCP(
+            userEmailAddress = "userea@liferay.com",
+            userFirstName = "userfn",
+            userLastName = "userln",
+            userScreenName = "usersn");
+
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase]: Use JSON macro to add users. See: https://issues.liferay.com/browse/LPS-130691

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 5]
```diff
@@ -1564,4 +1564,80 @@ definition {
...
+            userEmailAddress = "userea@liferay.com");
+			
+		Click(locator1 = "UserBar#USER_AVATAR_IN_NAVIGATION_USER_PERSONAL_BAR_PORTLET");	
+
+        Click(locator1 = "UserBar#USER_DASHBOARD");
+
+        AssertElementNotPresent(
+           key_nodeName = "Blank Site Name",
+           locator1 = "Treeview#NODE_ITEM");
+
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase]: Why you do the check here? Isn't it supposed that you have to check it in the Site and Asset Library's modal within the Content Dashboard?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 6]
```diff
@@ -1564,4 +1564,80 @@ definition {
 			title = "WC Title");
 	}
 
+	@description = "When the user clicks on My Sites or Asset Libraries tabs within the modal, the list of Sites or Asset Libraries to which the user has NO access is NOT displayed"
+    @priority = "5"
+    test SitePermissions {
+        property portal.acceptance = "true";
+
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase]: Please add tasks to divide the test and make it more readable. See: https://issues.liferay.com/browse/LPS-130691

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]

**Comment by CarlosBrichete** [comment]
Hey @RitaSilveira , add a description to the PR indicating the JIRA tickets affected and the motivation for this PR.

**Pull request merged**